packages/components/src/Popover/Popover.css

```diff
@@ -1,9 +1,19 @@
-:root {
-  --card--base-padding: var(--space-base);
```

Comment:
this is not referenced anywhere in this file or in Jobber. There is a variable for this in Card, actually highlighting the fragility of these living on `:root`.

Reply:
that's very strange

I wonder why we were using card variables in popover when it definitely doesn't contain cards by default. gonna take a look at the PR where we added that. maybe cards were doing something weird in Popovers?